tutorial/tutorial.md

```diff
@@ -0,0 +1,245 @@
+<!--
+SPDX-Copyright: Free Software Foundation Europe e.V.
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Tutorial: How to become REUSE compliant
+
+This tutorial explains the basic methods of how to make a software project REUSE
+compliant. By the end of this document, all your files will clearly have their
+copyright and licensing marked, and you will be able to verify this using a
+linter tool.
+
+Making your project REUSE compliant can be done in three simple steps:
+
+1. Choose and provide licenses
+2. Add copyright and license information to each file
+3. Confirm REUSE compliance
+
+For the purpose of this tutorial, we will assume that the directory of your
+project looks like this:
+
+```
+project/
+├── img/
+│   ├── cat.jpg
+│   └── dog.jpg
+├── src/
+│   └── main.c
+├── .gitignore
+├── Makefile
+└── README.md
+```
+
+If you would like to reproduce the steps in this tutorial on your own
+computer, you can clone the [example
+repository](https://github.com/fsfe/reuse-example). The branch `noncompliant`
+matches the structure above, while the `master` branch is the successful
+result of this repository.
+
+## 1. Choose and provide licenses
+
+The first thing you need to do is to [choose a
+license](https://choosealicense.com/). For this tutorial, we assume that you
+chose the GNU General Public Licence v3.0 or any later version (GPL). More
+than simply choosing a license, you need to put the license in your project
+directory.
+
+You find your license in the [SPDX License List](https://spdx.org/licenses/).
+SPDX is an open standard for communicating license and copyright information.
+Each license is uniquely identified by a shortform [SPDX
+identifier](https://spdx.org/ids). The SPDX identifier for your chosen license
+is GPL-3.0-or-later.
+
+<!-- TODO: Use tool instead -->
+
+You create a `LICENSES` directory in your project root which will contain all
+the licenses that you use in your project. You then download your license
+from the
+[license-list-data](https://github.com/spdx/license-list-data/tree/master/text)
+repository and put it in the `LICENSES` directory.
+
+## 2. Add copyright and license information to each file
+
+Now that you have a license, you need to indicate in the relevant files that
+these files fall under that license. You edit the comment header of
+`src/main.c` as such:
+
+```
+/*
+ * SPDX-Copyright: 2019 Jane Doe <jane@example.com>
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+```
+
+<!-- FIXME: Fix links -->
+
+The `SPDX-Copyright` tag records the publication years and copyright holder
+of the contents of the file. You can read more about [which publication years
+to use]() and [what copyright holders are]() in the FAQ.
+
+The `SPDX-License-Identifier` tag is followed by a [valid SPDX License
+Expression](https://spdx.org/specifications), typically just the SPDX
```

Comment:
I agree with changing the URL, but I'm not sure about the change from License Expression to License Identifier.

From the spec:

> SPDX License Expressions provides a way for one to construct expressions that more accurately represent the licensing terms typically found in open source software source code. A license expression could be a single license identifier found on the SPDX License List; a user defined license reference denoted by the LicenseRef-{idString}; a license identifier combined with an SPDX exception; or some combination of license identifiers, license references and exceptions constructed using a small set of defined operators (e.g., AND, OR, WITH and +). We provide the definition of what constitutes a valid an SPDX License Expression in this section. 

So it seems to me:

identifier = single tag
expression = combined tags (or single tag)

The fact that the tag `SPDX-License-Identifier:` says identifier instead of expression is confusing, though.

Reply:
Wait, I didn't read the suggestion quite right. There are three terms here, not two:

SPDX License Expression: Combined tags
SPDX License Identifier: Single tag
SPDX Identifier: References within the document

I think?

Reply:
That is how I understand it, yes. Summoning @jlovejoy and @kestewart for confirmation.